src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,75 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COMMA = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String inputFile = readFile(fromFileName);
+        writeToFile(inputFile, toFileName);
+    }
+
+    private String readFile(String fileName) {
+        File file = new File(fileName);
```

Comment:
you don't need to create a file explicitly here. Simply pass a file name to the FileReader constructor